app/code/core/Mage/Catalog/Model/Product/Attribute/Source/Countryofmanufacture.php

```diff
@@ -45,10 +45,15 @@ public function getAllOptions()
         if (Mage::app()->useCache('config') && $cache = Mage::app()->loadCache($cacheKey)) {
             $options = unserialize($cache);
         } else {
-            $collection = Mage::getModel('directory/country')->getResourceCollection();
+            if (Mage::app()->getStore()->isAdmin()) {
```

Comment:
Refactor suggestion:

            $collection = Mage::getModel('directory/country')->getResourceCollection();
            if (!Mage::app()->getStore()->isAdmin())
                  $collection->loadByStore();

This way the patch becomes additive only.

Reply:
ok, i update commit ! thanx